src/utils/syntax-highlight.js

```diff
@@ -83,8 +89,6 @@ async function importHighlightFileForLanguage(language) {
       } else {
         languageFile = await import(
           /* webpackChunkName: "highlight-js-[request]" */
-          // eslint-disable-next-line max-len
```

Comment:
I love the idea of making the set of supported languages configurable, but I'm running into issues testing this. I'm not sure if it's just a mistaken assumption on how it should work though.

My assumption is that the `VUE_APP_HLJS_LANGAGES` variable would be an opt-in feature to allow supporting additional languages not present in the default build and that the normal behavior should be unchanged if the flag is not provided—is that correct?

If so, there may be an issue with how this is implemented, because I'm finding that existing JavaScript code listings are no longer highlighted even when not specifying this env var (as one example, haven't tried others yet).

Reply:
I think this line might be the problematic one, since I believe this magic comment is what allowed us to dynamically reference the necessary syntax file for a given page as needed.

Reply:
To reproduce the issue I'm seeing:

Run `npm run docs:preview` and load http://localhost:8080/documentation/swift-docc-render/internals to see the js/html highlighted code listings

Then build your branch with `npm run build` and run it again with `DOCC_HTML_DIR=/path/to/swift-docc-render/dist npm run docs:preview` and load http://localhost:8080/documentation/swift-docc-render/internals and note that the listings are no longer highlighted, even without specifying the new `$VUE_APP_HLJS_LANGUAGES` var

Reply:
Good call, I had broken it with the last changes. Removing `\.js` from the regex fixed it, as that was adding `-js` to the name of the all the language files in the bundle. Now it correctly highlights the original set of languages and any additional ones if specified.

Reply:
Awesome, glad to find out it was such an easy fix!